presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaInternalFieldDescription.java

```diff
@@ -75,7 +76,12 @@
     /**
      * <tt>_key_length</tt> - length in bytes of the key.
      */
-    KEY_LENGTH_FIELD("_key_length", BigintType.BIGINT, "Total number of key bytes");
+    KEY_LENGTH_FIELD("_key_length", BigintType.BIGINT, "Total number of key bytes"),
+
+    /**
+     * <tt>_timestamp</tt> - offset timestamp
+     */
+    OFFSET_TIMESTAMP_FIELD("_timestamp", TimestampType.TIMESTAMP, "Offset Timestamp");
```

Comment:
Use `TimestampType createTimestampType(3)` instead of the deprecated `TimestampType.TIMESTAMP`.